src/components/project-works/Project.jsx

```diff
@@ -0,0 +1,65 @@
+import { Box, Typography, useMediaQuery } from "@mui/material";
+import React from "react";
+import { AiOutlineRight } from "react-icons/ai";
+
+import PrimaryButton from "../Buttons/PrimaryButton";
+import { useTheme } from "@emotion/react";
```

Comment:
does it import you want

import { useTheme } from "@mui/material";

Reply:
That is not an import I intended to use it is a mistake.